content/download/docker.en.md

```diff
@@ -9,7 +9,7 @@ layout: "os"
   
 #### Quick links
 
-[ [**GRASS GIS 8.0 (current)**](#GRASS-GIS-current) | [**GRASS 7.8 (legacy)**](#GRASS-GIS-old) | [**GRASS 8.2 (preview)**](#GRASS-GIS-devel) ]
+[ [**GRASS GIS 8.2 (current)**](#GRASS-GIS-current) | [**GRASS 7.8 (legacy)**](#GRASS-GIS-old) | [**GRASS 8.3 (preview)**](#GRASS-GIS-devel) ]
```

Comment:
Legacy version should be changed to 8.0, I would say.

Reply:
Originally, I thought the same, but @echoix raised a valid point in https://github.com/OSGeo/grass-website/pull/305#issuecomment-1129356403 regarding why to keep 7.8 for a while longer...

Reply:
On a PSC meeting some time in the past, we have discussed (_not approved_), under [Scenario 2](https://trac.osgeo.org/grass/wiki/Release/Schedule#Scenario2), a policy where we, as project, don't actually support any other releases than the current release:

> We have only one actively updated, maintained, and supported release series.

However, under this scenario, we do support people updating any older release series when they choose to do so:

> These versions (series/branches) are no longer actively updated, but are updated on demand, i.e., if you submit a patch to fix a bug, we will likely accept it and create a new release when there is enough changes accumulated.

Under this scenario, @neteler (based on latest commits and releases) is the person who is maintaining 7.8 as an individual contributor based on his own decision while the project stopped supporting 7.8 already. (This is obviously just good old do-ocracy, but expressed more clearly.) Under this scenario, we are just stopping support of 8.0 with the release of 8.2.0. And because there is no special treatment of the "one before current" release series, 7.8 and 8.0 are on the same level, i.e., not supported, although contributors can successfully push for a next maintenance release of either of those.

Under this scenario, given the above, it is @neteler's decision to keep 7.8 here or switch to 8.0 or, e.g., even support both, but don't have them listed here.

As I said, we didn't vote on this nor made any decision later as far as I recall.

I think this would be also different from just keeping links to old releases ("older installer versions"/"archived downloads") on the page as part of @echoix comment suggests. These all would not be maintained, but simply a couple of the last Windows installers would be linked (without the need for people to figure out the URL themselves).

Reply:
> Under this scenario, @neteler (based on latest commits and releases) is the person who is maintaining 7.8 as an individual contributor based on his own decision while the project stopped supporting 7.8 already.

Please note that in some Linux distributions G7.8 is included (see https://repology.org/project/grass/versions), e.g. Debian 11 (upcoming Debian 12, then with G8.0), Ubuntu 22.04 (upcoming Ubuntu 22.10, then with G8.0), etc. So "my" do-ocracy effort is to backport important stuff as needed even for G7.8. However, given my limited resources I'd also phase out the G7.8 support in the next months.
In any case, this is a discussion for grass-dev or grass-psc lists rather than in a pull request.